### ISSUE TITLE
File Extension not work with Viewable

### ISSUE THREAD

**Issue by Daniel-Dos** [mvc-spec/ozark#211]
when trying to use the file extension with the return type Viewable a 500 is displayed as shown in the screenshots: 

![eeror](https://user-images.githubusercontent.com/8139890/46568238-cd43bf00-c917-11e8-8b82-73c540d66c3a.png)

![error2](https://user-images.githubusercontent.com/8139890/46568241-d16fdc80-c917-11e8-8bc2-130cc4926ea7.png)

I made a small correction in the following PR, where to add another else if to Viewable, with that it started working for the 4 return types.

**Comment by chkal** [comment]
Good catch! And thanks for also creating the pull request to fix this issue. :+1:
